plugins/aws/index.js

```diff
@@ -0,0 +1,70 @@
+module.exports = {
+    $name: require('./package.json').name,
+    cliCommands: getCliCommands(),
+};
+
+function getCliCommands() {
+    return [
+        {
+            name: 'aws-configure',
+            param: '[access-key-id] [secret-access-key] [region]',
+            description: `Create/Overwrite ${getConfigFilePath('config', true)} and ${getConfigFilePath('credentials', true)}`,
+            action: execAws,
+            printAdditionalHelp() {
+                const {colorCmd, colorEmphasisLight, colorUrl, indent} = require('@brillout/cli-theme');
+                console.log(`${indent}${colorCmd('access-key-id')}        e.g. ${colorEmphasisLight('ABCDEFGHIJ0123456789')}`);
```

Comment:
maybe I could indent these even more?

Reply:
Looks like these are 4-space indents which is what Reframe's code uses so we're good.
Anyways I won't nitpick on indentation 😊